prelude/bigint.md

```diff
@@ -30,7 +36,7 @@ impl BigInt {
         }
 
         // the modulus.
-        let m = BigInt::from(2).pow(size.bits());
+        let m = BigInt::one() << size.bits();
```

Comment:
Please don't use shifts to encode exponentiation. That's a low-level bit manipulation trick, nothing we want to use in a specification.